charts/tidb-operator/values.yaml

```
@@ -129,6 +129,10 @@ controllerManager:
   # PodAnnotations will set template.metadata.annotations
   # Refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
   podAnnotations: {}
+  ## KubeClientQPS indicates the maximum QPS to the kubenetes api from client.
```

```suggestion
  ## KubeClientQPS indicates the maximum QPS to the kubenetes API server from client.
```

rename some `kubenetes api` to `kubenetes API server`?

---

OK.